korge/src/korlibs/korge/view/Text.kt

```diff
@@ -230,6 +230,10 @@ open class Text(
         return _textMetricsResult ?: error("Must ensure font is resolved before calling getGlyphMetrics")
     }
 
+    fun getGlyphMetricsForSpaceCharacter(): TextMetricsResult {
+        return font.getOrNull()?.getTextBoundsWithGlyphs(fontSize, " ", renderer, alignment) ?: error("Must ensure font is resolved before calling getGlyphMetrics")
+    }
```

Comment:
Since we are adding it, what about?

```kotlin
getGlyphMetricsForText(text: String)
```

So it is more generic. Or at least make it internal.